site/content/en/docs/Guides/metrics.md

```diff
@@ -191,7 +194,10 @@ kubectl apply -f ./build/grafana/
 Now we can install [grafana chart](https://github.com/helm/charts/tree/master/stable/grafana) from stable repository. (Replace `<your-admin-password>` with the admin password of your choice)
 
 ```bash
-helm install --wait --name grafana stable/grafana --version=5.0.13 --namespace metrics \
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+
+helm upgrade --install --wait grafana grafana/grafana --version=5.7.10 --namespace metrics \
```

Comment:
The same commands can be used to install and upgrade, so it has been modified to use upgrade.